llvm/test/MC/ARM/pcrel-vldr-diff-section.s

```diff
@@ -0,0 +1,14 @@
+@ RUN: not llvm-mc -filetype=obj -o /dev/null %s 2>&1 -triple=armv8.2a-eabi     | FileCheck %s
```

Comment:
same-section and diff-section tests can be in one file. `llvm/test/MC/ARM/pcrel-vldr-same-section.s` contains a lot of details that seem unrelated to the main purpose of the patch and should be reduced. There are also lots of instructions that may contain duplicates.

Reply:
How would you put them in the same test? Vldr_same/diff_section.s  have slightly different code, one compiles, the other one doesn't. If I merge them it will fail as a whole.

Reply:
As for the "unrelated details", I think you're referring to the checks of the addend values? And/or the symbol+imm checks?  We've done something similar in all the tests related to pcrel relocations. It was recommended to me as a good extra check by @smithp35 .  Here I just added a few more because we had a discussion on another review about whether an issue could be related to a pcrel instruction referring to a symbol before it rather than after. It wasn't but still I covered both cases from then on as to make it clearer.